### PR TITLE
[BACKLOG-8624] - Make Localization Service available by default in pr…

### PR DESCRIPTION
**Pull request by marcoslarsen**
[BACKLOG-8624] - Make Localization Service available by default in products (Add to Feature)

### CODE REVIEW

**Comment by marcoslarsen** [comment]
@gundisalwa @candentira @plagoa @francisco-guilherme  please review

**Comment by marcoslarsen** [comment]
@hudak please review

File: pentaho-karaf-features/src/main/resources/pentaho-karaf-features-standard.xml [hunk 1]
```diff
@@ -255,5 +256,8 @@
         <!-- <feature version="${cxf.karaf.version}">cxf-features-clustering</feature> -->
         <bundle>mvn:commons-collections/commons-collections/${commons.collections.version}</bundle>
     </feature>
-
+    <feature name="pentaho-i18n" version="${project.version}">
+      <bundle>mvn:pentaho/pentaho-i18n-bundle/${project.version}</bundle>
+      <bundle>mvn:pentaho/pentaho-i18n-webservice-bundle/${project.version}</bundle>
+    </feature>
```
Comment (graimundo) [inline pentaho-karaf-features/src/main/resources/pentaho-karaf-features-standard.xml]: bundle versions should be according the propertized version of the osgi bundle project: dependency.pentaho-osgi-bundles.revision

Check the following link for a list of existing properties in the future: http://devci.pentaho.com/job/master-snapshot/view/control/job/build-resources/ws/6.1-SNAPSHOT/version.properties/*view*/

**Comment by marcoslarsen** [comment]
@graimundo Thanks

**Pull request closed without merging**